float32 point_x
float32 point_y
float32 uncertainty_major
float32 uncertainty_minor
float32 uncertainty_orientation
